Jenkinsfile

```diff
@@ -95,7 +104,18 @@ def runRelease() {
             downloadToolsCert()
             print "Uploading version $version to Repo21"
             uploadCli(architectures)
-            distributeToReleases("jfrog-cli", version, "cli-rbc-spec.json")
+            stage("Distribute jfrog-cli to releases") {
```

Comment:
Distribute jfrog-cli to releases --> Distribute executables

(better to keep the stage titles short, because it probably ooks better in the Jenkins UI).